.github/CONTRIBUTING.md

```diff
@@ -142,7 +149,10 @@ builder failures, you may be experiencing a [test suite
 issue](https://github.com/zfsonlinux/zfs/issues?q=is%3Aissue+is%3Aopen+label%3A%22Test+Suite%22).
 There are also various [buildbot options](https://github.com/zfsonlinux/zfs/wiki/Buildbot-Options)
 to control how changes are tested.
-* All proposed changes must be approved by a ZFS on Linux organization member.
+* If your pull request has not yet passed the test suite,
+please submit it as a draft-PR instead.
```

Comment:
"draft PR" instead of "draft-PR", and likewise a couple times below

Reply:
Taking feedback into account (also from @behlendorf ), we could make it:
`please think about submitting it as a draft PR instead.`